### DISCUSSION TITLE
Iron release

### DISCUSSION THREAD

**Discussion by russkel** [eProsima/vulcanexus#140] [Q&A]
Hello,

Is a release for iron on the roadmap?

Kind regards,

Russ

**Comment by JLBuenoLopez** [comment]
Hi @russkel 

There is going to be a Vulcanexus Iron release in the next few days!

**Reply by russkel** [reply]
Thank you!

**Reply by JLBuenoLopez** [reply]
Hi @russkel,

Here you have the announcement of [Vulcanexus Iron release](https://discourse.ros.org/t/new-vulcanexus-iron-release/32579)

**Reply by russkel** [reply]
Thanks @JLBuenoLopez-eProsima, appreciate it!
